packages/components/src/components/ScalarMenu/ScalarMenu.vue

```diff
@@ -61,7 +61,7 @@ function close() {
     <DropdownMenu.Content
       align="start"
       :as="ScalarDropdownMenu"
-      class="max-h-radix-popper"
+      class="max-h-radix-popper z-context"
```

Comment:
this seems to set the submenu under the menu:
<img width="474" alt="image" src="https://github.com/user-attachments/assets/8b8e4e50-3f78-4dbe-8e74-00fd9b4fdb3f" />

Reply:
Ah we probably need it to use context plus